### ISSUE TITLE
Polar chart doesn't respect 'connectEnds: false'

### ISSUE THREAD

**Issue by izaleu** [highcharts/highcharts#8455]
#### Expected behaviour
Line series in polar chart should show the ends of the series

#### Actual behaviour
Line series in polar chart connects ends even though `connectEnds` is set to `false`

#### Live demo with steps to reproduce
[Example with sorted data
](http://jsfiddle.net/740Lbnmh/)
[Example with unsorted data
](http://jsfiddle.net/qy735gtr/) (Note the Highcharts error, even though the series looks correct)
#### Product version
 Highcharts JS v6.1.0 (2018-04-13)

#### Affected browser(s)
Version 66.0.3359.181 (Official Build) (64-bit) OSX High Sierra

#### Comments

- This only happens when our data is sorted by the x or y value. If the data is unsorted, it renders properly, but Highcharts throws [error 15](https://www.highcharts.com/errors/15) which is very spammy (because ReactJS, but still)
- Datasets that don't cross the vertical midline are unaffected [example](http://jsfiddle.net/m3obpctx/1/)
- I am totally willing to believe this is user error, but I've been mucking with this for days and haven't come up with a solution that isn't a hack

**Comment by pawelfus** [comment]
Hi @izaleu 

Thank you for reporting this issue, but this is not a bug. Let's start with unsorted data: actually you have sorted that dataset so result is what you expect. For example, sorted **descending** data you will get the same result: http://jsfiddle.net/BlackLabel/qy735gtr/2/

What really happens, is better to visualise using this demo: http://jsfiddle.net/BlackLabel/740Lbnmh/5/ (just removed most of the points). Ends are not connected, but it looks like because of values. Point x=64.18 is connected to the last point x=359.94. Point 359.94 is **not** connected to x=0.13. If you increase extremes you will see this better: http://jsfiddle.net/BlackLabel/740Lbnmh/7/ and http://jsfiddle.net/BlackLabel/740Lbnmh/9/

In short, it's not a user error, this dataset should look that way :)

**Comment by izaleu** [comment]
I'm a little confused, the first link you shared (with data sorted descending by X) is not the "same result" as the unsorted link I shared. In any case, I think I have a better grasp of how Highcharts is drawing the lines now, so thanks.

It looks like my options are A) Ignore the Highcharts error and leave the data unsorted to give the look I want, hope the error spam doesn't blow up the browser B) Turn off the lines and only use the dots C) split the data into two series as it crosses the vertical midline (ew)

**Comment by pawelfus** [comment]
@izaleu 

There is also another option: use scatter series with line. Scatter series doesn't require sorted data: http://jsfiddle.net/BlackLabel/qy735gtr/4/
